sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -70,6 +71,7 @@ def plot(self, *, include_values=True, cmap='viridis',
             Axes object to plot on. If `None`, a new figure and axes is
             created.
 
+        colorbar : matplotlib colorbar, default=True
         Returns
```

Comment:
If you check the [rendering of the documentation](https://102981-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.metrics.ConfusionMatrixDisplay.html#sklearn.metrics.ConfusionMatrixDisplay.plot) you see that you need a blank line before the "Returns" paragraph in order to keep the correct indentation.